src/components/Nav.js

```diff
@@ -49,9 +49,6 @@ const useStyles = makeStyles(theme => ({
   linkButton: {
     margin: theme.spacing(1, 1.5),
   },
-  toolbar: {
-    flexWrap: "wrap",
```

Comment:
Flex wrap seems to work better on current layout in mobile screen widths. However changes to navigation and collapsible navigation will need rework so I think it's ok no remove wrapping for now.

Reply:
I removed this because for some reasons with the new MUI, flexWrap is kind of covering other elements so some cypress tests have failed because of that.